### PR TITLE
add default log config

### PR DESCRIPTION
**Pull request by bxy4543**
The default log file is /root/.sealer.json or something else?

#464

### CODE REVIEW

File: logger/log.go [hunk 1]
```diff
@@ -149,6 +153,23 @@ func init() {
 	defaultLogger = NewLogger(3)
 }
 
+func Cfg(configFile string, debugMod bool, logFile string) {
+	var logCfg logConfig
+	viper.SetConfigType("json")
+	err := viper.Unmarshal(&logCfg)
+	if err != nil {
+		cfg, _ := json.Marshal(&logCfg)
```
Comment (justadogistaken) [inline logger/log.go:161]: ~/.sealer/config.json is just a string? not a concrete struct? what does the config look like?

File: logger/log.go [hunk 2]
```diff
@@ -362,22 +383,57 @@ func SetLogger(param ...string) {
...
-		fd, err := os.Open(c)
-		if err != nil {
-			fmt.Fprintf(common.StdErr, "Could not open %s for configure: %s\n", c, err)
-			os.Exit(1)
-		}
+		//configuration file does not exist, use the default configuration
+		if f, err := os.Stat(c); err != nil || f.IsDir() {
+			var (
+				loglevel   = 5
+				loggerfile = filepath.Join(common.GetHomeDir(), ".sealer", "sealer.log")
```
Comment (justadogistaken) [inline logger/log.go:390]: and how about we save log with more info? sealer-2011-1-1-13-13-13.log or something?

File: logger/log.go (hunk 1 above)
Comment (bxy4543) [reply logger/log.go:161]: {"TimeFormat":"2006-01-02 15:04:05","Console":{"level":"","color":true,"LogLevel":5},"File":{"filename":"/root/.sealer/sealer.log","append":true,"maxlines":1000000,"maxsize":1,"daily":false,"maxdays":-1,"level":"TRAC","permit":"0660","LogLevel":0}}
https://github.com/alibaba/sealer/blob/f7218f721e9ca413e1db9a05722a55a72952774b/logger/log_test.go#L56-L70

File: sealer/cmd/root.go [hunk 3]
```diff
@@ -51,28 +51,21 @@ func Execute() {
 
 func init() {
 	cobra.OnInitialize(initConfig)
-	rootCmd.PersistentFlags().StringVar(&rootOpt.cfgFile, "config", "", "config file (default is $HOME/.sealer.yaml)")
+	rootCmd.PersistentFlags().StringVar(&rootOpt.cfgFile, "config", "", "config file (default is $HOME/.sealer.json)")
```
Comment (fanux) [inline sealer/cmd/root.go:54]: I think we don't need to add a config file, just save logs in ${rootfs}/var/log/xxx.log.

Comment (bxy4543) [reply sealer/cmd/root.go:54]: If the configuration file does not exist, the default configuration is used. The default log file path is similar to /var/log/sealer/2006-01-01.log

**Review by fanux — APPROVED** [review]

**Pull request merged**